tests/src/main/java/net/neoforged/neoforge/oldtest/CreativeModeTabTest.java

```diff
@@ -117,27 +117,26 @@ private static ItemStack i(ItemLike item) {
     }
 
     private static void onCreativeModeTabBuildContents(BuildCreativeModeTabContentsEvent event) {
-        var entries = event.getEntries();
         var vis = TabVisibility.PARENT_AND_SEARCH_TABS;
         if (event.getTabKey() == LOGS) {
-            entries.putAfter(i(Blocks.ACACIA_LOG), i(Blocks.STRIPPED_ACACIA_LOG), vis);
-            entries.putAfter(i(Blocks.BIRCH_LOG), i(Blocks.STRIPPED_BIRCH_LOG), vis);
-            entries.putAfter(i(Blocks.DARK_OAK_LOG), i(Blocks.STRIPPED_DARK_OAK_LOG), vis);
-            entries.putAfter(i(Blocks.JUNGLE_LOG), i(Blocks.STRIPPED_JUNGLE_LOG), vis);
-            entries.putAfter(i(Blocks.OAK_LOG), i(Blocks.STRIPPED_OAK_LOG), vis);
-            entries.putAfter(i(Blocks.SPRUCE_LOG), i(Blocks.STRIPPED_SPRUCE_LOG), vis);
+            event.putAfter(i(Blocks.ACACIA_LOG), i(Blocks.STRIPPED_ACACIA_LOG), vis);
+            event.putAfter(i(Blocks.BIRCH_LOG), i(Blocks.STRIPPED_BIRCH_LOG), vis);
+            event.putAfter(i(Blocks.DARK_OAK_LOG), i(Blocks.STRIPPED_DARK_OAK_LOG), vis);
+            event.putAfter(i(Blocks.JUNGLE_LOG), i(Blocks.STRIPPED_JUNGLE_LOG), vis);
+            event.putAfter(i(Blocks.OAK_LOG), i(Blocks.STRIPPED_OAK_LOG), vis);
+            event.putAfter(i(Blocks.SPRUCE_LOG), i(Blocks.STRIPPED_SPRUCE_LOG), vis);
         }
 
         if (event.getTabKey() == STONE) {
-            entries.putBefore(i(Blocks.STONE), i(Blocks.SMOOTH_STONE), vis);
-            entries.putBefore(i(Blocks.GRANITE), i(Blocks.POLISHED_GRANITE), vis);
-            entries.putBefore(i(Blocks.DIORITE), i(Blocks.POLISHED_DIORITE), vis);
-            entries.putBefore(i(Blocks.ANDESITE), i(Blocks.POLISHED_ANDESITE), vis);
+            event.putBefore(i(Blocks.STONE), i(Blocks.SMOOTH_STONE), vis);
+            event.putBefore(i(Blocks.GRANITE), i(Blocks.POLISHED_GRANITE), vis);
+            event.putBefore(i(Blocks.DIORITE), i(Blocks.POLISHED_DIORITE), vis);
+            event.putBefore(i(Blocks.ANDESITE), i(Blocks.POLISHED_ANDESITE), vis);
         }
 
         // Adding this causes a crash (as it should) when opening the creative inventory
 //        if (event.getTabKey() == DAMAGED_SWORDS) {
-//            entries.putBefore(i(Items.WOODEN_SWORD), i(Items.WOODEN_SWORD), vis);
+//            event.putBefore(i(Items.WOODEN_SWORD), i(Items.WOODEN_SWORD), vis);
 //        }
```

Comment:
I know I wrote this, but I'm thinking it might be better to make this just a short-circuiting boolean condition statement instead, like
```java
if (false) {
    if (event.getTabKey() == DAMAGED_SWORDS) {
        entries.putBefore(i(Items.WOODEN_SWORD), i(Items.WOODEN_SWORD), vis);
        event.putBefore(i(Items.WOODEN_SWORD), i(Items.WOODEN_SWORD), vis);
    }
}
```
that way any future changes to this code path can be caught for this optional element. Ideally, a better method can be written to catch this with a game test or something

Reply:
I think at some point, we are gonna delete the old test files. (Though Maty probably has plans) The putBefore and i(...) are used in the modern CreativeTabOrderTest so it should be covering what this old test does already. I'll put it in if statement but if there's a use case here not covered by CreativeTabOrderTest, lmk but seems like I got all the event methods and validation checks